save-plugins/warn-plugin/README.md

```diff
@@ -83,7 +83,7 @@ testNameSuffix = "Test" # (default value)
 ```
 
 When executed from project root (where `save.propertes` is located), SAVE will cd to `rootDir` and discover all files
```

Comment:
please write to main readme: "Your application should return 0 in case of successful processing, otherwise save will not be able to determine correct result"

Reply:
Done